index.js

```diff
@@ -0,0 +1,94 @@
+const form = document.getElementById('form');
+
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const container = document.querySelector('.books-li');
+
+function saveLocalStorage(book, id) {
+  const bookId = { book, id };
+  const arr = localStorage.getItem('pdfs') ? JSON.parse(localStorage.getItem('pdfs')) : [];
+  arr.push(bookId);
+
+  localStorage.setItem('pdfs', JSON.stringify(arr));
+}
+
+function deleteFromLocalStorage(id) {
```

Comment:
Book item is not deleted exactly because when user clicks on `Remove` button and it gets hide from the UI but when user refreshes the page the same book item is displayed again. KIndly make sure the local storage is updated every time user adds or removes the book item.